run.py

```diff
@@ -274,10 +292,11 @@ def sanitize_router_address(address):
     server['serv_port_up'] = int(os.getenv("SERVER_0_PORTUP", 1700))
     server['serv_port_down'] = int(os.getenv("SERVER_0_PORTDOWN", 1700))
     server['serv_enabled'] = True
+    # Downlink data is by default enabled by packet forwarder. When required override default setting
     if(os.getenv('SERVER_0_DOWNLINK', "false")=="true"):
-      server['serv_down_enabled'] = True
-    else:
       server['serv_down_enabled'] = False
+    else:
+      server['serv_down_enabled'] = True
```

Comment:
I don't believe this change is correct.

`if(os.getenv('SERVER_0_DOWNLINK', "false")=="true"):` check if the environment variable is set to `true` and originally we then set the configuration to True. Now you check if it is true and set the config to False if it is.

Reply:
Agreed, I mixed up the logic.

Reply:
It might be good to consider adding the GW_AUTOQUIT_THRESHOLD parameter to the script.
This does not confilict with the principles that only one server shall be controlling downlink of a gateway.

Reply:
Yes, I do want to include the rest of your changes, so if you revert the downlink changes on your side and re-open this PR I can merge it.